### PR TITLE
[Gekidou] Show login screen when selecting a previous server

### PR DESCRIPTION
**Pull request by enahum**
#### Summary
When selecting a server without an active session from the server list, take the user directly to the login screen and bypass the Add Server screen.

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-43640

```release-note
NONE
```

### CODE REVIEW

**Review by matthewbirtch — APPROVED** [review]
Nice! This is a great improvement I think.

**Review by larkox — APPROVED** [review]

File: app/screens/home/channel_list/servers/servers_list/server_item/server_item.tsx [hunk 1]
```diff
@@ -195,8 +197,22 @@ const ServerItem = ({highlight, isActive, server, tutorialWatched}: Props) => {
...
+        setSwitching(true);
+        const result = await doPing(server.url);
+        if (result.error) {
+            alertServerError(intl, result.error as ClientErrorProps);
+            setSwitching(false);
+            return;
+        }
+        const data = await fetchConfigAndLicense(server.url, true);
+        if (data.error) {
+            alertServerError(intl, result.error as ClientErrorProps);
```
Comment (jfrerich) [inline app/screens/home/channel_list/servers/servers_list/server_item/server_item.tsx:211]:
```suggestion
            alertServerError(intl, data.error as ClientErrorProps);
```

**Review by jfrerich — CHANGES_REQUESTED** [review]
I have one change request and then I'll approve!

**Review by jfrerich — APPROVED** [review]
Fastest turnaround time ever!  Nice Job!

**Pull request merged**